packages/twenty-front/src/modules/object-record/cache/utils/getRecordEdgeFromRecord.ts

```diff
@@ -10,11 +11,26 @@ export const getRecordEdgeFromRecord = <T extends ObjectRecord>({
   objectNameSingular: string;
   record: T;
 }) => {
+  const nestedRecord = Object.fromEntries(
```

Comment:
So we were having problems with edges/node and plain arrays ?

Reply:
yes, it was not recursively done